Update one_way_anova.ipynb

pxr687 opened the pull request:
- Added some illustrations of how SNSQGMD and the F-stat change as a function of the distance between the group means

- Some other small suggested edits

pxr687 committed:
Update one_way_anova.ipynb

- Added some illustrations of how SNSQGMD and the F-stat change as a function of the distance between the group means

- Some other small suggested edits

matthew-brett commented on oneway/one_way_anova.Rmd:
````diff
@@ -1,23 +1,33 @@
     name: python3
 ---
 
+```{python}
````
For some reason I prefer this boilerplate lower down - maybe to separate the introduction, not-code part, from the code part.   What do you think?

pxr687 replied:
Assuming I interpreted this correctly, in the 'simulation graphs as images' commit below I've moved the 'import numpy as np (etc).' section down into the first code cell (where the raw data is read in).

matthew-brett commented on oneway/one_way_anova.Rmd:
```diff
@@ -68,30 +83,128 @@ Difference).
+SNSQGMD  will be larger when larger groups have means with bigger deviations from the 
+overall mean.
+
+The three graphs below use some simulated data to show how SNSQGMD changes
```
I do like the simulation idea - but I worry that it puts a big chunk of into the notebook.   I know that the user can't see it in the rendered page, but they can when they get the notebook itself.   What I'd prefer to do here, is to move the simulations off to another page, and point to it from here.  What do you think?

pxr687 replied:
Yes, I see what you mean. On the ANOVA page, could we just display an image file of the graphs from the simulation? I think it'd be good to have the graphs on the main page, to allow the reader to visualize whats being said in the text about SNSQGMD and the F-stat e.g. how they vary as a function of the group means? 

We could then point to a separate page, containing the code for the simulation, for those who are interested in how the graphs were generated?

pxr687 replied:
In the 'simulation graphs as images' commit below I have removed the simulation code from the main page. I've displayed a JPG of the graphs on the main page, and have pointed to a separate document containing the actual code.

What do you think regarding this?

matthew-brett commented on oneway/one_way_anova.Rmd:
````diff
@@ -68,30 +83,128 @@ Difference).
+```{python}
+# HIDDEN CELL
+
+# a function to compute SNSQGMD
````
Isn't it a bit confusing using another function for the SNSQGMD here?

pxr687 replied:
Yes it is! But I was intending all the simulation code cells to be hidden - I didn't think about hiding the cells only applying to the rendered page....

matthew-brett reviewed (commented):
A few reflections - thanks - in general I like the changes.

pxr687 committed:
simulation graphs as images

- added image files to display the simulation graphs on the main page, rather than running the code on the main page itself

- created a separate notebook with the simulation code, and pointed to it from the main page, for interested readers
